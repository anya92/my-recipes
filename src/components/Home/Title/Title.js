import React from 'react';
import styled from 'styled-components';

const StyledTitle = styled.h1`
  text-align: center;
  font-size: 3rem;
  margin: 20px 0 10px 0;
  span {
    font-weight: 400;
  }
`;

const Title = () => <StyledTitle><span>my</span>Recipes</StyledTitle>;

export default Title;
