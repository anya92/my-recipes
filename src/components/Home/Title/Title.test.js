import React from 'react';
import { shallow } from 'enzyme';

import Title from './Title';

test('should render Title correctly', () => {
  const wrapper = shallow(<Title />);
  expect(wrapper).toMatchSnapshot();
});
