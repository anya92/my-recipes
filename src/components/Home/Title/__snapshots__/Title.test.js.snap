// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Title correctly 1`] = `
<styled.h1>
  <span>
    my
  </span>
  Recipes
</styled.h1>
`;
